packages/story-editor/src/components/header/buttons/test/publish.js

```diff
@@ -127,6 +133,14 @@ describe('PublishButton', () => {
     fireEvent.click(publishButton);
 
     expect(showPriorityIssues).toHaveBeenCalledTimes(1);
+
+    const publishButtons = screen.queryAllByRole('button', {
```

Comment:
I know in the past, and with other dialogs / modals they can sometimes take time before rendering. It might be advantageous to use `await screen.findAllByRole`